### PR TITLE
Added USONE

### PR DESCRIPTION
**Pull request by SJPMurphy**

### CODE REVIEW

**Comment by SJPMurphy** [comment]
My last request was knocked back as we didn't have a 'working' website - we have it up and working now

File: registry.json [hunk 1]
```diff
@@ -3372,5 +3372,20 @@
...
+    "poolId": "576e0390ac54e3751afdaa8aa256f41b84a6324006f23991f6fdefb7",
+    "social": {
+      "facebook": "",
+      "twitter": "as_usone",
+      "website": "https://usas.one",
+      "reddit": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "us_as_one",
```
Comment (leo42) [inline registry.json:3387]: please use a personal telegram handle. No groups or pages

**Review by leo42 — DISMISSED** [review]

File: registry.json (hunk 1 above)
Comment (leo42) [reply registry.json:3387]: .

**Review by leo42 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 1 above)
Comment (SJPMurphy) [reply registry.json:3387]: Personal telegram is @SJPMurphy

**Pull request merged**